packages/ckb-sdk-rpc/src/defaultRPC.ts

```diff
@@ -40,7 +40,7 @@ const defaultRPC: CKBComponents.Method[] = [
   {
     name: 'getLiveCell',
     method: 'get_live_cell',
-    paramsFormatters: [paramsFmts.toOutPoint],
+    paramsFormatters: [paramsFmts.toCellOutPoint],
```

Comment:
Isn't params is a `OutPoint`?

Reply:
> Isn't params is a `OutPoint`?

Damn it, comes from the outdated document.

Reply:
😮

Reply:
updated in https://github.com/nervosnetwork/ckb-sdk-js/pull/167/commits/29f323b2d5534afaaf75dccd5fe3fcd498bc784d